appwrite/appwrite
# Fix 952 patch user prefs event

## What does this PR do?

Adds an event for that fires when user preferences are updated.

## Test Plan

- Create test user
- Add webhook for `users.update.prefs`
- Update test user prefs
- Verify webhook call is successful.

Also adding E2E test coverage for this event.

## Related issues
#952 

## Todo

- [x] Create `users.update.prefs` event
- [x] Fix E2E test

## UI
![updateprefsevent](https://user-images.githubusercontent.com/9708641/110014713-208ce800-7cf1-11eb-8a4a-87e81c844937.png)


### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

yes
